### PR TITLE
pyglet GL context for SimpleImageViewer case

### PR DESCRIPTION
**Pull request by shashikg**
@pzhokhov @christopherhesse

### CODE REVIEW

File: gym/envs/classic_control/rendering.py [hunk 1]
```diff
@@ -51,23 +51,26 @@ def get_display(spec):
...
     """
     screen = display.get_screens() #available screens
     config = screen[0].get_best_config() #selecting the first screen
     context = config.create_context(None) #create GL context
 
-    return pyglet.window.Window(width=width, height=height, display=display, config=config, context=context)
+    if viewer_type == 'Viewer':
+        return pyglet.window.Window(width=width, height=height, display=display, config=config, context=context)
+    elif viewer_type == 'SimpleImageViewer':
+        return pyglet.window.Window(width=width, height=height, display=display, config=config, context=context, vsync=False, resizable=True)
```
Comment (pzhokhov) [inline gym/envs/classic_control/rendering.py:65]: how about window_kwargs argument rather, and call `pyglet.window.Window(width=width, height=height, display=display, config=config, context=context, **window_kwargs)` here?

Comment (pzhokhov) [reply gym/envs/classic_control/rendering.py:65]: OTOH I wonder if there is a strong reason for vsync=False and resizabble=True in SimpleImageViewer in the first place

Comment (shashikg) [reply gym/envs/classic_control/rendering.py]: I am not sure but maybe because we are fixing the window size, so we set the vsyns=False.

Comment (shashikg) [reply gym/envs/classic_control/rendering.py]: > how about window_kwargs argument rather, and call `pyglet.window.Window(width=width, height=height, display=display, config=config, context=context, **window_kwargs)` here?

done!

**Pull request merged**